en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md

```diff
@@ -179,6 +179,9 @@ If you want to update the secret token dynamically without restarting the server
     }
     ```
 
+!!! Tip
+    You can use the WSO2 API Controller, **apictl** to update the secret token dynamically. For more information, see [Update HashiCorp configuration]({{base_path}}/install-and-setup/setup/api-controller/managing-integrations/managing-integrations-with-ctl/#update-hashicorp-configuration).
```

Comment:
Why do we keep saying '**apictl**' ? Are we using it as an acronym?
The other pages (APIM pages) doesn't seem to use it.. If we want to use it, it has to be,

"You can use the WSO2 API Controller (APICTL) to update the secret token dynamically. For more information, see [Update HashiCorp configuration]({{base_path}}/install-and-setup/setup/api-controller/managing-integrations/managing-integrations-with-ctl/#update-hashicorp-configuration)."

@Mariangela / @ruthryi - FYI

Reply:
+1

@npamudika , what do you think?

Reply:
+1 for this

Reply:
+1

In the Getting Started page we have used as "WSO2 API Controller(CTL)" and all the other pages we have used like "WSO2 API Controller, apictl". Shall we change all the place to "WSO2 API Controller (APICTL)"? @uvindra WDYT?

Reply:
Referring to https://apim.docs.wso2.com/en/latest/learn/api-controller/getting-started-with-wso2-api-controller/, https://apim.docs.wso2.com/en/latest/learn/api-controller/importing-apis-via-dev-first-approach/, and the other pages

Reply:
+1

Reply:
@wasuradananjith will you be able to do these changes along with the other fixes you are doing?

Reply:
Sure. Created an issue [1] to keep track of this. Can @SanojPunchihewa do the appropriate changes to this PR as well?

[1] https://github.com/wso2/docs-apim/issues/3156

Reply:
Also, there are several places as "API Controller" in our documentation. Are we going to change that as well to "WSO2 API Controller" or "WSO2 API Controller (APICTL)"?

Reply:
We don't need to keep saying "WSO2 API Controller (APICTL)". The normal standing is, if APICTL is the acronym, we need to first introduce the acronym (at the top of the page). From then on, we can use APICTL instead of WSO2 API Controller.

From another point of view, I believe we also wanted to establish 'apictl' in the same way as 'kubectl'. If that is the case, perhaps we should use 'apictl' consistently throughout (like they have done in the Kubernetes documentation). Right now, we seem to use several references: WSO2 API Controller, CTL Tool, apictl (we use this very casually in the doc).

Reply:
> Sure. Created an issue [1] to keep track of this. Can @SanojPunchihewa do the appropriate changes to this PR as well?
> 
> [1] #3156

Fixed with 3b5162b

Reply:
> We don't need to keep saying "WSO2 API Controller (APICTL)". The normal standing is, if APICTL is the acronym, we need to first introduce the acronym (at the top of the page). From then on, we can use APICTL instead of WSO2 API Controller.
> 
> From another point of view, I believe we also wanted to establish 'apictl' in the same way as 'kubectl'. If that is the case, perhaps we should use 'apictl' consistently throughout (like they have done in the Kubernetes documentation). Right now, we seem to use several references: WSO2 API Controller, CTL Tool, apictl (we use this very casually in the doc).

+1, Let's do like that. On any page when the first time we refer to API Controller let's introduce the abbreviation.